introduction.tex

```diff
@@ -52,41 +41,52 @@ \section{Methodology}
 France can reprocess used \gls{UOX} and \gls{MOX} to
 produce \gls{MOX} from reprocessed plutonium and depleted uranium (tails).
 
-%%% PASSIVE PHRASE WHAT TO DO
 After obtaining the \gls{UNF} inventory of all \gls{EU} in 2050, the second
 simulation runs where the \gls{UNF} inventory is reprocessed and fabricated
 as fuel for the newly deployed \gls{SFR} reactors.
-\gls{SFR} per model after the ASTRID reactor.
-ASTRID-type \glspl{SFR} make up for the decommissioned capacity
+\glspl{SFR} are modeled after the ASTRID breeder reactor \cite{varaine_pre-conceptual_2012}.
+The ASTRID-type \glspl{SFR} make up for the decommissioned capacity
 of \glspl{LWR} in France, to remain a constant installed capacity of $66,000$ MWe up to 2160.
-It is assumed that ASTRID-type reactors use \gls{MOX} fuel created from 11\% reprocessed plutonium
-and 89\% tails and burns the \gls{MOX} fuel to approximately 100 GWdth/t.
-The high burnup allows breeding of plutonium.
 Eventually, the  \gls{MOX} created from recycled \gls{MOX}
 fuels the entire fleet of 110 \glspl{SFR}.
 
+All the scripts and data used
+in this paper are available in \cite{bae_arfc/transition-scenarios:_2017}.
+
+\subsection{\Cyclus}
+\Cyclus is an agent-based fuel cycle simulation framework, meaning that
+each reactor, reprocessing plant, and fuel fabrication plant is modeled as agents. At each timestep (one month),
+agents put out their bids for materials (supply and/or demand) and exchange
+with one another using a market-like mechanism called the 
+dynamic resource exchange \cite{huff_fundamental_2016}.
+Each material item has a quantity, composition, name, and unique identifiers
+for output analysis. 
+A \Cyclus input file contains archetypes, which are fuel cycle facilities with
+pre-defined parameters, that are deployed in the simulation as \texttt{facility} agents.
+An \texttt{Institute} agent deploys \texttt{facility} agents
+according to a user-defined deployment scheme at pre-defined timesteps.
+The \texttt{Institute} agent is part of a \texttt{Region} agent,
+which can contain multiple \texttt{Institute} agents.
+
+For example, `France' would be a \texttt{Region} agent,
+that may contain two \texttt{Institute} agents \glspl{LWR}
+and \glspl{SFR}. The \texttt{Institute} agents would then deploy
+\glspl{LWR} and \glspl{SFR} agents, respectively, according to a pre-defined deployment
```

Comment:
Don't name the institution the same thing as an agent. Call it LWR institution if you must. Anyway, the deploy institution should be able to deploy more than one kind of archetype.... not sure why you need separate institutions for each technology.

Reply:
oh, makes. But I think there is some benefit showing that a region can have multiple institutions..?
Maybe `reactors` and `fuel cycle facilities`?

Reply:
Institutions are supposed to model real world institutions (Areva, Westinghouse, Exelon...)